analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/HashesShouldHaveUnpredictableSalt.cs

```diff
@@ -202,12 +190,16 @@ public void ByteArrayCases(byte[] passwordBytes)
 
         var shortArray = new byte[] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 };
         rng.GetBytes(shortArray);
-        new PasswordDeriveBytes(passwordBytes, shortArray);                                         // FIXME Non-compliant
+        new PasswordDeriveBytes(passwordBytes, shortArray);                                         // Noncompliant
 
         var longEnoughArray = new byte[] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15 };
         rng.GetBytes(longEnoughArray);
         new PasswordDeriveBytes(passwordBytes, longEnoughArray);                                    // Compliant
 
+        var evenLongerArray = new byte[] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16 };
+        rng.GetBytes(evenLongerArray);
+        new PasswordDeriveBytes(passwordBytes, evenLongerArray);                                    // Compliant
```

Comment:
Sorry I was not clear here - please add it without passing it after from the random number generator to be sure it raises for hardcoded arrays.